test/pages/security.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import SecurityPage from '../../src/pages/security';
+import '../__mocks__/intersectionObserverMock';
+import { createGeneralPageData } from '../__fixtures__/page';
+
+const mockData = createGeneralPageData();
+describe('Privacy page', () => {
```

Comment:
I think this should say `Security Page` 🙂

Other than that the PR seems ready to be merged